src/Service/Lambda/tests/Integration/LambdaClientTest.php

```diff
@@ -55,6 +55,19 @@ public function testInvoke(): void
         self::assertSame('$LATEST', $result->getExecutedVersion());
     }
 
+    public function testInvokeBatchAsync(): void
+    {
+        $client = $this->getClient();
+
+//        $input = new InvocationRequest([
```

Comment:
If you have time I would need help to understand how you test this
- Run tests
- How I can lead `http://localhost:9001` to return something. Or shall I prefer using mocking ?